mathesar/api/serializers/db_types.py

```diff
@@ -9,6 +9,8 @@ class DBTypeSerializer(serializers.Serializer):
 
     def to_representation(self, db_type):
         return {
-            "id": db_type.value,
+            # TODO solve db type casing holistically
+            # https://github.com/centerofci/mathesar/issues/1036
+            "id": db_type.value.upper(),
```

Comment:
I'm resolving this comment because, as of now, the final diff in this PR does not show any changes to `db_types.py`. I'm not sure what happened either, but things look fine to me now.